### ISSUE TITLE
CMake error when no GRIB found

### ISSUE THREAD

**Issue by slayoo** [gnudatalanguage/gdl#792]
```
-- Looking for grib_index_new_from_file in GRIB_LIBRARIES-NOTFOUND
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
GRIB_LIBRARIES
    linked by target "cmTC_77552" in directory /home/slayoo/Downloads/gdl-1.0.0-rc.3/build/CMakeFiles/CMakeTmp

CMake Error at /usr/share/cmake-3.13/Modules/CheckLibraryExists.cmake:56 (try_compile):
  Failed to configure test project build system.
Call Stack (most recent call first):
  CMakeModules/FindGrib.cmake:18 (check_library_exists)
  CMakeLists.txt:923 (find_package)


-- Configuring incomplete, errors occurred!
See also "/home/slayoo/Downloads/gdl-1.0.0-rc.3/build/CMakeFiles/CMakeOutput.log".
See also "/home/slayoo/Downloads/gdl-1.0.0-rc.3/build/CMakeFiles/CMakeError.log".
```

**Comment by alaingdl** [comment]
but did not my change few hours ago solved that ? https://github.com/gnudatalanguage/gdl/pull/791/commits/2bc315843f8dd0693b4924ad7a7d9bfe82c19ceb#diff-9b0ca30900780f9a688796ca278e0fa2 in PR #791 ?

**Comment by slayoo** [comment]
Indeed, thanks!
Thinking out loud: should we do 1.0.0-rc.4 because of it? (probably not...)

**Comment by alaingdl** [comment]
Quite a busy week. I started in fact to clean-up the formatting of outputs (messages)
in  CMakeLists.txt but it take time. For homogeneity and clarity. In fact the formatting
are not that clear until I found here https://gitlab.kitware.com/cmake/cmake/-/issues/19418 a clear mention of two facts :
* unformatted lines are line-wrapped to 77 character line width (FATAL & co)
* if started with a space, they are supposed to be formatted, then no rule "77" apply
